setup_ngrok.py

```
@@ -0,0 +1,143 @@
+                    print(f"   Payload URL: {public_url}/webhook/github")
+                    print("   Content type: application/json")
+                    print("   Events: Push events")
+                    print()
+                    return public_url, process
+                else:
+                    print("❌ No tunnels found")
+                    return None, process
+            else:
+                print("❌ Could not connect to ngrok API")
+                return None, process
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)